src/main/java/base/TestBase.java

```diff
@@ -31,7 +31,7 @@ public static ChromeOptions chromeOption() {
         options.setExperimentalOption("prefs", chromePrefs);
         options.setCapability(CapabilityType.ACCEPT_INSECURE_CERTS, true);
         options.addArguments("--start-maximized");
-        options.addArguments("--incognito");
+//        options.addArguments("--incognito");
```

Comment:
don't leave commented code, either enable it or remove it, so if you don't want to run headless remove it.
or if you want to make it more complex you can do if condition with boolean and but the headless by true or false in the .properties file